packages/shared/src/UI/contexts/PageUIProvider.tsx

```diff
@@ -25,11 +26,25 @@ interface MaskUIRootProps extends React.PropsWithChildren<{}> {
     fallback?: React.ReactNode
 }
 
+const persister = createSyncStoragePersister({
+    storage: window.localStorage,
+})
+// We don't persist all queries but only those have the first key starts with '@@'
```

Comment:
does this means query key like `@@normal-data` will be stored and `normal-data` will not?

Reply:
> does this means query key like `@@normal-data` will be stored and `normal-data` will not?

Yes, just a magic mark.